src/utility/components.js

```diff
@@ -493,129 +542,225 @@ export class Chant {
-        mode = authenticMode;
-        rating += 0.12;
-        modeDescription += `Plagal repercussio detected, the third most repeated note is '${authenticRepercussio}' (${counts[authenticRepercussio]} times).\n`;
-      } else if (sortedCounts[2] === plagalRepercussio) {
-        mode = plagalMode;
-        rating += 0.12;
-        modeDescription += `Plagal repercussio detected, the third most repeated note is '${plagalRepercussio}' (${counts[plagalRepercussio]} times).\n`;
-      } else if (sortedCounts[3] === authenticRepercussio) {
-        /** If one of the two expected repercussio notes is the fourth most repeated note: 6% */
-        mode = authenticMode;
-        rating += 0.06;
-        modeDescription += `Authentic repercussio detected, with the fourth most repeated note is '${authenticRepercussio}' (${counts[authenticRepercussio]} times).\n`;
-      } else if (sortedCounts[3] === plagalRepercussio) {
-        mode = plagalMode;
-        rating += 0.06;
-        modeDescription += `Plagal repercussio detected, the fourth most repeated note is '${plagalRepercussio}' (${counts[plagalRepercussio]} times).\n`;
+      if (sortedCounts[1] === plagalRepercussioPitch && sortedCounts[0] === finalisPitch) { // case (ii)
+        plagalRepercussioRating += 0.75;
+        repercussioPlagDesc.innerHTML = `Plagal repercussio '${plagalRepercussioPitch.toUpperCase()}' of mode '${plagalMode}'\
+          is the second most repeated note (${counts[plagalRepercussioPitch]} times) `;
+        repercussioPlagDesc.innerHTML += `after finalis '${finalisPitch.toUpperCase()}'. `;
+      } else if (sortedCounts[1] === plagalRepercussioPitch && sortedCounts[0] !== finalisPitch) { // case (iii)
+        plagalRepercussioRating += 0.5;
+        repercussioPlagDesc.innerHTML = `Plagal repercussio '${plagalRepercussioPitch.toUpperCase()}' of mode '${plagalMode}'\
+          is the second most repeated note (${counts[plagalRepercussioPitch]} times) `
+        repercussioPlagDesc.innerHTML += `after a note other than finalis '${finalisPitch.toUpperCase()}'. `;
+      } else if (sortedCounts[2] === plagalRepercussioPitch) { // case (iv)
+        plagalRepercussioRating += 0.25;
+        repercussioPlagDesc.innerHTML = `Plagal repercussio '${plagalRepercussioPitch.toUpperCase()}' of mode '${plagalMode}'\
+          is the third most repeated note (${counts[plagalRepercussioPitch]} times). `;
+      } else if (sortedCounts[3] === plagalRepercussioPitch) { // case (v)
+        plagalRepercussioRating += 0.125;
+        repercussioPlagDesc.innerHTML = `Plagal repercussio '${plagalRepercussioPitch.toUpperCase()}' of mode '${plagalMode}'\
+          is  the fourth most repeated note (${counts[plagalRepercussioPitch]} times). `;
       } else {
-        modeDescription += `No expected repercussio note is repeated more than 3 times.\n`;
+        repercussioPlagDesc.innerHTML = `Unable to detect plagal mode from repercussio. `;
```

Comment:
`<plaga_repercussio_pitch>` is not among the most repeated notes. Therefore, there is **0%** probability of being in plagal mode.